.gitignore

```diff
@@ -1,3 +1,24 @@
-**/node_modules
-
 .DS_Store
+node_modules
+/dist
+package-lock.json
+
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
To be honest I don't know what these files are and could you please explain where they come from? I don't have too much knowledge on Vue but I know sometimes it would generates/require a lot of strange stuff.

Reply:
Like with the previous stuff, I believe these are all precautionary to prevent files that are created by the editor you use from being pushed to git. I do not know all of them, I assume they are just common files endings for different editors.

Reply:
I removed unnecessary formats. Generally it would be added until we really need those.

Reply:
I looked it up and most of them are for Microsoft Visual Studio. Not sure what .ntvs and .sw are for.

Reply:
Okay, seems like it is because of different editors. I would take a look on that in the future review if anyone use different editors. But anyway, it's not a big trouble.